templates/taxi/car_list.html

```diff
@@ -11,8 +11,11 @@ <h1>
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
+          {{ car.id }} 
+          <a href="{% url "taxi:car-detail" pk=car.id %} ">
```

Comment:
Ensure that the URL tag uses single quotes consistently for better readability. Change `{% url "taxi:car-detail" pk=car.id %}` to `{% url 'taxi:car-detail' pk=car.id %}`.